src/data_collection/helpers.py

```diff
@@ -0,0 +1,9 @@
+from django.conf import settings
+from django.db.models import Q
+from rest_framework.response import Response
+from .models import Data
+
+def logger(cls, data):
+	new_data = Data(data_type=cls, log=data)
```

Comment:
if the field name is `data_type`, can you name this variable to be `data_type` instead of `cls`